docs/configuration/setup-static-loabalancer-ip.md

```diff
@@ -0,0 +1,46 @@
+# Setup a static IP for loadbalancer
```

Comment:
This document looks like it's a different PR and has slipped in here - is it?

Reply:
good catch - thanks